CalibTracker/SiStripChannelGain/plugins/SiStripGainFromData.cc

```diff
@@ -1038,7 +1039,7 @@ SiStripGainFromData::algoAnalyze(const edm::Event& iEvent, const edm::EventSetup
   
       for(Trajectory::RecHitContainer::const_iterator rechit = transRecHits.begin(); rechit != transRecHits.end(); ++rechit)
          if ((*rechit)->isValid()) ndof += (*rechit)->dimension();  
-         ndof -= 5;
```

Comment:
@mmusich

This piece of code is not used anywhere. Please ignore it.

Cheers,
Alessandro.


> On 11 Jul 2016, at 11:29, Marco Musich <notifications@github.com> wrote:
> 
> In CalibTracker/SiStripChannelGain/plugins/SiStripGainFromData.cc <https://github.com/cms-sw/cmssw/pull/15022#discussion_r70226662>:
> 
> > @@ -1038,7 +1039,7 @@ SiStripGainFromData::algoAnalyze(const edm::Event& iEvent, const edm::EventSetup
> >    
> >        for(Trajectory::RecHitContainer::const_iterator rechit = transRecHits.begin(); rechit != transRecHits.end(); ++rechit)
> >           if ((*rechit)->isValid()) ndof += (*rechit)->dimension();  
> > -         ndof -= 5;
> @dimattia <https://github.com/dimattia> @boudoul <https://github.com/boudoul> ping
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15022/files/8213e295092f8d3da1c9a54c1c90a73248d8f4c5#r70226662>, or mute the thread <https://github.com/notifications/unsubscribe/AIK-7gPTCbKtwgR3M2Ga62zShEFhT2dUks5qUgzvgaJpZM4JAT8a>.
>